easybuild/main.py

```diff
@@ -323,7 +323,7 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
     categorized_paths = categorize_files_by_type(orig_paths)
 
     # command line options that do not require any easyconfigs to be specified
-    no_ec_opts = [options.aggregate_regtest, options.new_pr, options.regtest, options.update_pr, search_query]
+    no_ec_opts = [options.aggregate_regtest, options.new_pr, options.preview_pr, options.regtest, options.update_pr, search_query]
```

Comment:
line too long, needs wrapping...

Reply:
Also, can't we use `no_ec_opts or new_update_pr` below, and avoid including `options.new_pr`, `options.update_pr` and `options.preview_pr` in `no_ec_opts` (since it's wrong, sematically)

Reply:
actually, an update_pr could just add or remove a patch, without any easyconfig in the command line...?

Reply:
OK, sure, but it doesn't make much sense for `--new-pr` and `--preview-pr` (although in theory you could also only use a patch file there)